src/components/settings/Account/Billing/index.tsx

```diff
@@ -74,7 +72,7 @@ export const Billing: React.FC<BillingProps> = ({ account }) => {
       {
         plan,
         feedback,
-        useTrial: !!claimedGrowth && validPayment,
+        useTrial: !account?.trialPeriodEndedAt,
```

Comment:
Can you quickly explain this logic change?

Reply:
Feels like it's very different from the prior logic, and there's not a lot of accompanying changes. Curious to know how you arrived at the new logic

Reply:
Yeah the orignal if can be broken into 2 parts:
1. claimedGrowth - this used to check to see if the browser has a localhost variable saved that gets set when the user originally claimed the growth package. This logic had a bug where if you logged into a different account on the same browser you wouldn't be given growth free trial. Now we just check if the trial period was claimed directly on the account which is the same thing but account-scoped
2. validPayment - this is validation logic that should happen server-side so it's not necessary to have here (I figured less code client-side the better). I could've done the same thing for the `claimedGrowth` variable (it does happen server side) but I didn't wanna change the behaviour too much from what it is now. The claimedGrowth logic occurs in other places in the billing flow too but validPayment is much rarer so it made sense to remove it from this scope

TL;DR - the old logic had a bug with sharing accounts on the same browser and unecessary business-logic client-side